instrumentation/internal/internal-lambda/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/internal/lambda/LambdaInstrumentationModule.java

```diff
@@ -28,8 +32,17 @@ public boolean defaultEnabled(ConfigProperties config) {
   }
 
   @Override
-  public boolean isIndyModule() {
-    return false;
+  public void injectClasses(ClassInjector injector) {
```

Comment:
I don't think having any `invokedynamic`-based instrumentation/proxies is a good idea for the lambda instrumentation: The reason is that we unecessarily risk recursions for the invokedynamic bootstrapping if lambdas are used in there in some unfortunate places, even if this is not the case right now.

I feel we should rather simply move those two classes to the agent bootstrap package, lambda instrumentation imo is special enough to justify this. Alternatively, we could directly inject the classes via `injectedClassNames()`.

Reply:
Good point, until moving those to the bootstrap package is further discusses I've switched to using `injectedClassNames()` in 45ead47